[doc] Refine the deployment documentation (#11361)
apache/dolphinscheduler: docs/docs/en/guide/installation/pseudo-cluster.md

## Modify Configuration

After completing the preparation of the basic environment, you need to modify the configuration file according to the

> environment you used.Env configuration can be changed in the following ways, export <ENV_NAME>=<VALUE>, The configuration files are both in directory `bin/env` and named `install_env.sh` and `dolphinscheduler_env.sh`.

Reviewer: ```suggestion
environment you used. Change the environment configurations via `export <ENV_NAME>=<VALUE>`. The configuration files are located in directory `bin/env` as `install_env.sh` and `dolphinscheduler_env.sh`.
```

# ---------------------------------------------------------
# Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
ips="localhost"

> sshPort=${sshPort：-"22"}

Reviewer 2: ```suggestion
sshPort="22"
```

# ---------------------------------------------------------
# Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
ips="localhost"

> # Modify Configurationexport sshPort="22"

Reviewer 2: Not this place, I mean we should maybe add some sentences in L79 or L144, to tell users they can change env variable by `export <ENV_NAME>=<VALUE>`

Reviewer 2: Others LGTM

> # Modify Configurationexport export sshPort="22"

Reviewer: We could remove this comment as we have already had it above.

Reviewer 2: Agree with that
